### ISSUE TITLE
validation_errors() opening and closing tag arguments

### ISSUE THREAD

**Issue by feerspreads** [trongate/trongate-framework#113]
I'm not sure if it's my setup or not, but when I try to implement the following:

`validation_errors('<div class="errors">', '</div>')`

In an otherwise working block of code ( while going through the task manager learning path on the site), it returns nothing; no output, no errors, nothing.

Tested using the json function and I get an empty string.

But,
`validation_errors()`
works as expected

Was something changed in the system, but not in the documentation?

**Comment by trongate** [comment]
Hi, It's a glitch.  Fear not.  I'll have it fixed as a priority.  Thank
you. - DC

**Comment by feerspreads** [comment]
Don't know if any of this helps, but I was fiddling around with the validation helper and it looks like it might have something to do with some kind of filter?

I tried to echo out the $opening_html on the first line of the function and it wouldn't output anything with brackets. Text without brackets went through fine.

Also the function wouldn't get past the first IF with this expression:
`(gettype($closing_html == 'boolean'))`

but made it through when I replaced it with this:
`($closing_html === false)`
